### PR TITLE
[K8SPXC-762] Validating webhook not accepting scale operation

### PR DESCRIPTION
**Pull request by defbin**
[K8SPXC-762](https://jira.percona.com/browse/K8SPXC-762) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

File: e2e-tests/validation-hook/run [hunk 1]
```diff
@@ -12,22 +12,42 @@ create_infra $namespace
...
+fi
+
+if ! output=$(kubectl 2>&1 wait --for=condition=ready pxc/simple-pxc --timeout=1m); then
+	echo "ERROR: ${output}"
+	exit 1
+fi
+
+scale_cmd_err='admission webhook "validationwebhook.pxc.percona.com" denied the request: json: unknown field "replicas"'
+
+if ! output=$(kubectl 2>&1 scale pxc/simple-pxc --replicas 1); then
```
Comment (delgod) [inline e2e-tests/validation-hook/run:45]: lets run real scale operation
```suggestion
if ! output=$(kubectl 2>&1 scale pxc/simple-pxc --replicas 2); then
```
and later check if it is really scaled

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| upgrade-haproxy | passed |
| scaling | passed |
| storage | passed |
| scheduled-backup | passed |
| recreate | passed |
| big-data | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| init-deploy | passed |
| upgrade-proxysql | passed |
| security-context | passed |
| demand-backup | passed |
| smart-update | passed |
| self-healing-chaos | failed |
| limits | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| affinity | passed |
| pitr | passed |
| one-pod | passed |
| self-healing-advanced-chaos | failed |
| operator-self-healing | passed |
| auto-tuning | passed |
| upgrade-consistency | passed |
| operator-self-healing-chaos | failed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup-encrypted-with-tls | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/993/commits/29e63e3710f5cb80eb25ede91e58eb3546f07e50
image: `perconalab/percona-xtradb-cluster-operator:PR-993-29e63e37`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scaling | passed |
| upgrade-haproxy | passed |
| scheduled-backup | passed |
| haproxy | passed |
| storage | passed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| self-healing-chaos | passed |
| limits | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| affinity | passed |
| one-pod | passed |
| self-healing-advanced-chaos | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| operator-self-healing | passed |
| users | passed |
| operator-self-healing-chaos | passed |
| tls-issue-self | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/993/commits/736102ba09d372b5c982d143e763236a49520844
image: `perconalab/percona-xtradb-cluster-operator:PR-993-736102ba`

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | passed |
| haproxy | passed |
| scheduled-backup | passed |
| init-deploy | passed |
| limits | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| scaling | passed |
| tls-issue-self | passed |
| scaling-proxysql | passed |
| tls-issue-cert-manager | passed |
| security-context | passed |
| tls-issue-cert-manager-ref | passed |
| recreate | passed |
| validation-hook | passed |
| big-data | passed |
| restore-to-encrypted-cluster | passed |
| storage | passed |
| demand-backup | passed |
| self-healing | passed |
| pitr | passed |
| self-healing-chaos | passed |
| demand-backup-encrypted-with-tls | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/993/commits/736102ba09d372b5c982d143e763236a49520844
image: `perconalab/percona-xtradb-cluster-operator:PR-993-736102ba`

**Review by egegunes — APPROVED** [review]

**Pull request merged**